src/main/scala/uk/gov/hmrc/apiplatform/modules/events/applications/domain/models/Actor.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apiplatform.modules.events.applications.domain.models
+
+/** Actor refers to actors that triggered an event
+  */
+sealed trait Actor
+
+object Actors {
+
+  /** A third party developer who is a collaborator on the application for the event this actor is responsible for
+    * triggering
+    *
+    * @param id
+    *   the developers email address at the time of the event
+    */
+  case class Collaborator(email: LaxEmailAddress) extends Actor
+
+  /** A gatekeeper stride user (typically SDST)
+    *
+    * @param id
+    *   the stride username of the gatekeeper user who triggered the event on which they are the actor
```

Comment:
Looking at code in third-party-application, the loggedInUser.userFullName is used, rather than the username, i.e. "Firstname Lastname" rather than "firstname.lastname".

Reply:
Changed to username in case class and adjusted comment.

Reply:
As discussed, changing the case class will affect existing events in the database

Reply:
And back again :-)